srcpkgs/Aegisub/template

```diff
@@ -0,0 +1,48 @@
+# Template file for 'Aegisub'
+pkgname=Aegisub
+version=3.2.2
+revision=1
+build_style=gnu-configure
+configure_args="--disable-update-checker $(vopt_with alsa) $(vopt_if fftw)
+$(vopt_if openal) $(vopt_if portaudio) $(vopt_if pulseaudio)"
+hostmakedepends="automake gettext-devel intltool libtool pkg-config"
+makedepends="boost-devel libcurl-devel fontconfig-devel freetype-devel
+hunspell-devel icu-devel libass-devel libffms2-devel LuaJIT-devel
+wxWidgets-devel $(vopt_if alsa alsa-lib-devel) $(vopt_if fftw fftw-devel)
+$(vopt_if openal libopenal-devel) $(vopt_if portaudio portaudio-devel)
+$(vopt_if pulseaudio pulseaudio-devel)"
+short_desc="Tool for creating and modifying subtitles"
+maintainer="Rasmus Thomsen <rasmus.thomsen@protonmail.com>"
+license="BSD-3-Clause, MIT"
+homepage="http://www.aegisub.org"
+distfiles="https://github.com/Aegisub/Aegisub/archive/v${version}.tar.gz"
+checksum=62757dd491455268a240f983b59734a801cc2e899039a7493deeaf5e24a61dcd
+
+build_options="alsa fftw openal portaudio pulseaudio"
+build_options_default="alsa fftw pulseaudio"
+
+desc_option_alsa="Enable support for alsa"
+desc_option_fftw="Enable support for rendering of audio waveforms/spectrum"
+desc_option_openal="Enable support for openal"
+desc_option_portaudio="Enable support for portaudio"
+desc_option_pulseaudio="Enable support for pulseaudio"
+
+post_extract() {
+	# Fix shebang to use system luajit instead of vendored minilua
+	sed -e \
+		"s:../vendor/luajit/src/host/minilua:/usr/bin/luajit:" \
+		-i tools/respack.lua
+}
+
+pre_configure() {
+	autoreconf -fi
+}
+
+do_build() {
+	make LDFLAGS="-pthread $LDFLAGS" ${makejobs}
```

Comment:
i believe this can be replaced by just adding

LDFLAGS="-pthread" to the global variables